datavec/datavec-python/src/main/java/org/datavec/python/PythonException.java

```diff
@@ -0,0 +1,7 @@
+package org.datavec.python;
+
+public class PythonException extends Exception {
+    public PythonException(String message){
```

Comment:
Add more constructors for more flexibility. We also need javadoc explaining what will typically throw this exception. Reference classes using @see  at the top.